primitives/sgx/crypto/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "substratee-sgx-crypto"
+version = "0.1.0"
```

Comment:
Is there a specific reason why this is `0.1.0`, whereas the rest of the worker is at `0.8.0` ?

Reply:
nope, this was just the default, when I created a new crate via de IDE. WIll update that